evi check/Form1.cs

```diff
@@ -0,0 +1,650 @@
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2GhostOrb.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3GhostOrb.Checked = false;
+                        }
+                        break;
+
+                    case "EMF5":
+                        if (group == 1)
+                        {
+                            this.Evi1EMF5.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2EMF5.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3EMF5.Checked = false;
+                        }
+                        break;
+
+                    case "GhostWriting":
+                        if (group == 1)
+                        {
+                            this.Evi1GhostWriting.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2GhostWriting.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3GhostWriting.Checked = false;
+                        }
+                        break;
+
+                    case "FingerPrints":
+                        if (group == 1)
+                        {
+                            this.Evi1FingerPrints.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2FingerPrints.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3FingerPrints.Checked = false;
+                        }
+                        break;
+
+                    case "SpiritBox":
+                        if (group == 1)
+                        {
+                            this.Evi1SpiritBox.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2SpiritBox.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3SpiritBox.Checked = false;
+                        }
+                       // this.Evi1SpiritBox.Checked = false;
+                        break;
+                }
+            }
+
+        }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        private void Evi1Cold_click(object sender, EventArgs e)
+        {
+
+            if (this.Evi1Cold.Checked == true)
```

Comment:
The `== true` here is redundant. 
```suggestion
            if (this.Evi1Cold.Checked)
```